tox.ini

```diff
@@ -59,6 +60,68 @@ commands =
     test: pytest --cov={env:PY_MODULE} --cov-report=term-missing:skip-covered --doctest-modules {posargs:src/{env:PY_MODULE}}
 
 
+##
+# Flake8 linting
+##
+
+[testenv:flake8]
+
+description = run Flake8 (linter)
+
+basepython = python3.7
+
+skip_install = True
+
+# Pin pydocstyle to version 3: see https://gitlab.com/pycqa/flake8-docstrings/issues/36
+deps =
+    flake8-bugbear==19.8.0
+    #flake8-docstrings==1.4.0
+    #flake8-import-order==0.18.1
+    #flake8-pep3101==1.2.1
```

Comment:
This PR is already pretty noisy, so not piling on with these.

Reply:
fair :)